test/e2e/e2e_test.go

```diff
@@ -325,9 +325,17 @@ func verifyResourcesProvisionedForSignup(t *testing.T, awaitility *wait.Awaitili
 		UserAccountStatus: userAccount.Status,
 	}
 	_, err = hostAwait.WaitForMasterUserRecord(mur.Name,
-		wait.UntilMasterUserRecordHasConditions(provisioned()),
+		wait.UntilMasterUserRecordHasConditions(provisioned(), provisionedNotificationCRCreated()),
 		wait.UntilMasterUserRecordHasUserAccountStatuses(expectedEmbeddedUaStatus))
 	assert.NoError(t, err)
+
+	notification, err := hostAwait.WaitForNotification(userAccount.Name + "-provisioned")
+	assert.NoError(t, err)
```

Comment:
```suggestion
	require.NoError(t, err)
```

Reply:
fixed as part of: 4afd56d532ea1db81994c09a35511c8bc26e2b79